Makefile

```
@@ -494,6 +495,70 @@ cli-test-snap:
+#     lib/scrub_agent_configs.py
+#     README.md                        (short usage)
+#
+# Ships as dist/defenseclaw-macos-$(VERSION)-$(GOOS)-$(GOARCH).tar.gz.
+#
+# Overrides: GOOS/GOARCH cross-compile the gateway.
+BUNDLE_GOOS  ?= darwin
+BUNDLE_GOARCH ?= $(shell uname -m | sed 's/x86_64/amd64/;s/aarch64/arm64/')
+BUNDLE_NAME  := defenseclaw-macos-$(VERSION)-$(BUNDLE_GOOS)-$(BUNDLE_GOARCH)
+BUNDLE_DIR   := $(DIST_DIR)/$(BUNDLE_NAME)
+
+packaging-macos-bundle:
```

checkmake (0.3.2)

[warning] Target body for "packaging-macos-bundle" exceeds allowed length of 5 lines (34).

(maxbodylength)